lua/timer.lua

```diff
@@ -37,22 +37,21 @@ function __stop_internal_timer(func)
 end
 
 __Timer_timer_processing = __Timer_timer_processing or false
-state.state:set_var("timer_enabled", 1)
+state.state.timer_enabled = 1
 __Timer_timer_process = function(seconds_per_tick)
 	if __Timer_timer_processing then
 		return 0
 	end
 	__Timer_timer_processing = true
-	local time = dm.world:get_var("time")
 	for func, timeData in __Timer_timers do
 		if timeData.executing == true then
 			continue
 		end
-		if over_exec_usage(0.85) then
+		if _exec.time / _exec.limit > 0.85 then
 			sleep()
 		end
```

Comment:
since this isn't such a low value as 100ms anymore, you may want to instead use tick_usage as a measure instead.
Otherwise, this'll probably end up eating all of the tick time.

Will probably need to look at implementing a library to handle performance since over_exec_usage isn't a super reliable indicator anymore to prevent the server from freezing up. It wasn't perfect before but now it definitely won't be any useful anymore.